test-resources/geo/presence-data-schema.json

```diff
@@ -0,0 +1,39 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Ably Asset Tracking Presence Data",
+  "type": "object",
+  "properties": {
+    "type": {
+      "type": "string",
+      "description": "The type of client in the presence channel.",
+      "enum": ["subscriber", "publisher"]
```

Comment:
@QuintinWillison it looks like we've not fully implemented our [naming policy for enum values](https://github.com/ably/ably-asset-tracking-android/issues/207) in the SDK, those values should be `"SUBSCRIBER", "PUBLISHER"` :disappointed: If we want to fix it we'll probably need to do it soon because this is a change that will break the communication between different versions of the SDK. I'll create an issue for that.

Reply:
Here's the issue for changing those enum values in the schema https://github.com/ably/ably-asset-tracking-common/issues/15. 
I've also created issues for each of our supported platforms (they're mentioned in this common issue).

Reply:
Thanks for managing that. We should, perhaps, be prioritising [that issue](https://github.com/ably/ably-asset-tracking-common/issues/15) and its children. FYI, @jamienewcomb, @kavalerov